src/Jira/Api.php

```diff
@@ -198,6 +198,16 @@ public function editIssue($issue_key, array $params)
 	}
 
 	/**
+	 * Gets attachment upload limit.
+	 *
+	 * @return array
+	 */
+	public function getAttachmentsMeta()
```

Comment:
Sorry for confusion is this fine?
`/**
	 * Gets attachment meta information.
	 *
	 * @return array
	 * @since  2.0.0
	 */
	public function getAttachmentsMetaInformation()`